### PR TITLE
PipTest: Upgrade expected pyparsing and six dependencies

### PR DESCRIPTION
**Pull request by fviernau**
Signed-off-by: Frank Viernau <frank.viernau@here.com>

### CODE REVIEW

**Comment by sschuberth** [comment]
Nit: Please indicate in the commit message *why* this upgrade needs to be done. To me, simply adding "expected" by saying "PipTest: Upgrade the expected pyparsing dependency to version 2.4.3" would be enough, as this explains that this is needed to make the test pass again.

File: analyzer/src/main/kotlin/managers/GoMod.kt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+package com.here.ort.analyzer.managers
```
Comment (sschuberth) [inline analyzer/src/main/kotlin/managers/GoMod.kt]: This file was apparently accidentally added.

**Comment by sschuberth** [comment]
pyparsing was upgraded yet again to version 2.4.4, and also six needs upgrading now, so I've boldly amended your commit @fviernau.

**Comment by fviernau** [comment]
Merge as the only test failing is unrelated (GradleKotlinScriptTest).

**Pull request merged**